GOVERNANCE.md

```diff
@@ -0,0 +1,246 @@
+| Right/responsibility | Description | 
+|---|---|
+| Decision making with respect to their algorithm  | Algorithm maintainers can partipate in the decision making process by vetoing changes and casting votes with regard to proposed changes to their algorithm. This does not extend to proposed changes to the common framework and API. | 
+| Maintenance | They are responsible for maintaining the code and documentation for their algorithm, including bug fixes, unit testing, coding style, compliance with the common API, docstrings, documentation and tutorials notebooks. | 
+| Support | They are the first point of contact for users and other contributors for all questions, issues and proposals regarding their algorithm. |
+
+#### Appointment
+
+The contributor who contributes an algorithm is automatically appointed as its first maintainer. If they can no longer fulfil their maintenance responsibilities, maintainers are expected to resign. 
+
+The maintainer can appoint another contributor as maintainer. No vote is required. An algorithm can have multiple maintainers. 
+
+When applicable, maintainers are listed in each file as the `__author__` string.
+
+### Core developers
+
+Core developers are contributors who have shown that they are dedicated to the continued development of the project through ongoing engagement with the community. They have shown they can be trusted to maintain sktime with care.
+
+Current core developers are listed in the [core-developers team](https://github.com/orgs/sktime/teams/core-developers/members) within the sktime organisation on GitHub.
+
+#### Rights and responsibilities
+
+| Right/responsibility | Description | 
+|---|---|
+| Direct access | Being a core developer allows contributors to more easily carry on with their project related activities by giving them direct access to the project’s repository. |
+| Issue/PR management | Core developers are responsible for reviewing and managing issues and pull requests. This includes commenting on issues, reviewing code contributions, merging approved pull requests, and closing issues once resolved. |
+| Decision making | They can partipate in the decision making process by vetoing changes and casting votes. |
+| Nomination | They can nominate new core developers, CoC committee members and CC members. | 
+
+#### Appointment
+New core developers can be nominated by any current core developer. Once they have been nominated, there will be a vote by the current core developers.
+
+Voting on new core developers is one of the few activities that takes place on the project's private chat. While it is expected that most votes will be unanimous, a 2/3 majority of the cast votes is enough. The vote needs to be open for at least one week.
+
+Core developers that have not contributed to the project (commits or GitHub comments) in the past 12 months will be asked if they want to become *emeritus core developers* and give up their rights and responsibilities until they become active again.
```

Comment:
what is the process to "become active again" - is it the same as a new appointment?